polaris-react/src/components/PolarisTestProvider/PolarisTestProvider.tsx

```diff
@@ -65,13 +65,18 @@ export function PolarisTestProvider({
 
   const stickyManager = useMemo(() => new StickyManager(), []);
 
+  const featuresConfig = useMemo(
+    () => ({...features, polarisSummerEditions2023: false}),
```

Comment:
`polarisSummerEditions2023` was missing from the `FeaturesContext` and it was causing some tests to fail.